pilot/pkg/credentials/kube/secrets.go

```diff
@@ -26,13 +26,11 @@ import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
-	"k8s.io/apimachinery/pkg/runtime"
+	sa "k8s.io/apiserver/pkg/authentication/serviceaccount"
```

Comment:
Should we depend on apiserver ?

Reply:
It is not k/k, this lib is aimed to be used